include/clang/Interpreter/CppInterOp.h

```diff
@@ -173,6 +173,9 @@ namespace Cpp {
   /// Checks if the passed value is an enum type or not.
   bool IsEnumType(TCppType_t type);
 
+  /// Returns vector of enums present in given scope
+  std::vector<std::string> GetEnums(TCppScope_t scope);
```

Comment:
```suggestion
  void GetEnums(TCppScope_t scope, std::vector<std::string>& Result);
```

We should change the signature in this way.